src/cls/rgw/cls_rgw_types.h

```diff
@@ -112,10 +113,11 @@ struct rgw_bucket_dir_entry_meta {
     ::encode(content_type, bl);
     ::encode(accounted_size, bl);
     ::encode(user_data, bl);
+    ::encode(accounted_entry, bl);
     ENCODE_FINISH(bl);
   }
   void decode(bufferlist::iterator &bl) {
-    DECODE_START_LEGACY_COMPAT_LEN(5, 3, 3, bl);
+    DECODE_START_LEGACY_COMPAT_LEN(6, 3, 3, bl);
```

Comment:
Is it necessary to change the `v` of `DECODE_START_LEGACY_COMPAT_LEN(v, compatv, lenv, bl)` from 5 to 6?
The `v` is different from `struct_v `~

Reply:
@ZVampirEM77 
Thanks for your suggestion.
The v is diffrent from struct_v,  but it looks like that the value of v and struct_v are always same in ceph codes.  Do you want me both change them from 6 to 5 ?

Reply:
@ZVampirEM77 

thanks for your suggestion. 

 I have modified the code with your review.